components/plan-build/bin/hab-plan-build.sh

```diff
@@ -2006,7 +2008,7 @@ _generate_artifact() {
   mkdir -pv "$(dirname "$pkg_artifact")"
   rm -fv $tarf $xzf $pkg_artifact
   $_tar_cmd -cf $tarf $pkg_prefix
-  $_xz_cmd --compress -6 --threads=0 $tarf
+  $_xz_cmd --compress -${HART_COMPRESSION_LEVEL} --threads=0 $tarf
```

Comment:
Wouldn't we want this ENV var to be prefixed with `HAB_` to match the idioms for other env vars?
https://www.habitat.sh/docs/reference/#environment-variables

Reply:
That makes sense. Are you ok with `HAB_HART_COMPRESSION_LEVEL` or just `HAB_COMPRESSION_LEVEL`. The latter seems not specific enough to me

Reply:
Either seems fine by mean. Maybe someone from @habitat-sh/habitat-core-maintainers has an opinion or idiom that will make one naming approach more appealing.

Reply:
I agree with @jaym.  `HAB_HART_COMPRESSION_LEVEL` feels like the best choice there, at least to me.